src/r2mm/launching/instructions/instructions/loader/CustomInstructions.ts

```diff
@@ -1,31 +1,39 @@
 import GameInstructionGenerator from "../GameInstructionGenerator";
 import Game from "../../../../../model/game/Game";
 import Profile from "../../../../../model/Profile";
-import * as path from 'path';
 import { GameInstruction } from "../../GameInstructions";
 import { DynamicGameInstruction } from "../../DynamicGameInstruction";
+import path from '../../../../../providers/node/path/path';
 
 export class CustomInstructions extends GameInstructionGenerator {
-    private readonly instructions: GameInstruction;
+    private readonly instructions: () => GameInstruction;
 
-    constructor(instructions?: GameInstruction) {
+    constructor(instructions?: () => GameInstruction) {
```

Comment:
Is the change to function form purely for optimization or does it serve some other purpose?